<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New game</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_P1">
     <property name="text">
      <string>Player &amp;1 name:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_P1</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_P1"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_P2">
     <property name="text">
      <string>Player &amp;2 name:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_P2</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_P2"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_P1</tabstop>
  <tabstop>lineEdit_P2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_P1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>updateOKButtonState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_P2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>updateOKButtonState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateOKButtonState()</slot>
 </slots>
</ui>
